### ISSUE TITLE
Yellow Ribbon MVP - Findings for Accessibility Staging Review 

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8785]
**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

This epic will contain the accessibility staging review findings issue tickets.

Findings currently exist in [this comment on the a11y spot check from April 6](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7344#issuecomment-609955111).

## Point of Contact

**VFS Point of Contact:** Josh Kim

@1Copenut ready for your review 2020-05-05

**Comment by 1Copenut** [comment]
@jenstrickland I agree with all of your comments from the [a11y spot check](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7344#issuecomment-609955111). I have just one addition and one proposed tweak to your markup pattern for the description list cards:

1. When users click a pagination link, we should be returning focus to the "Displaying 1-10 of 51 results" `<h2>`. Currently focus is staying with the pagination link that was clicked.
2. I ran your proposed card pattern through an axe check and W3C validation. Axe was throwing an error with `role="header"` on the `<dt>` and the validator didn't like it either. I worked with it a bit, and moved the heading into a div with the associated role and aria-level. This validates against axe and the W3C validator for HTML5:

```html
<dl>
  <dt>
    <div aria-level="3" role="heading">
      <dfn class="sr-only">School name</dfn>
      Abilene Christian University
    </div>
  </dt>
  <dd>
    <dfn class="sr-only">City, State</dfn>
    Abilene, <abbr title="Texas">TX</abbr>
  </dd>
  <dd>
    <dfn>Maximum Yellow Ribbon funding amount (per student, per year)</dfn>
    All tuition and fees not covered by Post-9/11 GI Bill benefits
  </dd>
  <dd>
    <dfn>Funding available for</dfn>
    All eligible students</dd>
  <dd>
    <dfn>School website</dfn>
    <a href="www.acu.edu" rel="noreferrer noopener">www.acu.edu</a>
  </dd>
  <dd>
    <dfn>Degree type</dfn>
    All
  </dd>
  <dd>
    <dfn>School or program</dfn>
    All
  </dd>
  <!-- More cards here -->
</dl>
```

**Comment by jenstrickland** [comment]
@1Copenut It's supposed to be `role="heading"` — typo on my part!

**Comment by MarciMcGuireGCIO** [comment]
**Sprint Planning 38 note:** While there are other defects open from post-MVP review, this is the only 508-3 defect needed to close out the Staging accessibility review: https://github.com/department-of-veterans-affairs/va.gov-team/issues/8416
